docker/yb-voyager-docker

```diff
@@ -54,8 +54,31 @@ for var_name in "${variables[@]}"; do
   fi
 done
 
+flags_to_store=("--source-ssl-cert"
+"--source-ssl-key"
+"--source-ssl-root-cert"
+" --source-ssl-crl"
+"--target-ssl-root-cert"
+"--target-ssl-crl"
+"--target-ssl-cert"
+"--target-ssl-key"
+"--source-replica-ssl-cert"
+"--source-replica-ssl-crl"
+"--source-replica-ssl-key"
+"--source-replica-ssl-root-cert")
+
 exported_vars="${exported_vars} -e ORACLE_HOME=/usr/lib/oracle/21/client64 -e LD_LIBRARY_PATH=/usr/lib/oracle/21/client64/lib -e PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/go/bin:/usr/lib/oracle/21/client64/bin"
 
+# if command is like yb-voyager export data. The command name is export data.
+command_name=${argv[0]}-${argv[1]}
+# if argv[2] is "from" or "to" then command name is like import data from source or import data to target
+if [[ ${argv[2]} == "from" || ${argv[2]} == "to" ]]
+then
+	command_name=${argv[0]}-${argv[1]}-${argv[2]}-${argv[3]}
+fi
+file_name="/tmp/docker-metadata-${command_name}.txt"
```

Comment:
let's use a variable name that is more specific? say `command_metadata_file_name` ?